broker-cli/proto/broker.proto

```diff
@@ -794,6 +854,117 @@ service OrderService {
       get: "/v1/orders"
     };
   }
+  /**
+   * GetTradeHistory returns all completed and executing orders and all accepted and executed fills.
+   *
+   * ```javascript
+   * const deadline = new Date().setSeconds(new Date().getSeconds() + 5)
+   * const blockOrder = orderService.getTradeHistory({}, { deadline }, (err, res) => {
+   *   if (err) return console.error(err)
+   *   console.log(res)
+   * })
+   * ```
+   *
+   * ```shell
+   * > sparkswap order trade-history
+   * ```
+   *
+   * > The above command will retrieve the trades on the Broker and will print:
+   *
+   * ```javascript
+   * {
+   *   "orders": [
+   *     {
+   *       "orderId": "Ky13eTmuhZY-NIbqIng_in3D",
+   *       "orderStatus": "PLACED",
```

Comment:
placed orders shouldn't show up in trade history, right?

Reply:
Oh oops, this is leftover from testing.